src/install/installer.ts

```diff
@@ -91,37 +95,29 @@ async function validateCache(packagePath: string, browsersPath: string, linksDir
   }
 
   // 2. Delete all unused browsers.
-  let downloadedBrowsers = (await fsReaddirAsync(browsersPath)).map(file => path.join(browsersPath, file));
-  downloadedBrowsers = downloadedBrowsers.filter(file => browserPaths.isBrowserDirectory(file));
+  let downloadedBrowsers = (await fsReaddirAsync(registryDirectory)).map(file => path.join(registryDirectory, file));
+  downloadedBrowsers = downloadedBrowsers.filter(file => isBrowserDirectory(file));
   const directories = new Set<string>(downloadedBrowsers);
-  for (const browserPath of usedBrowserPaths)
-    directories.delete(browserPath);
+  for (const browserDirectory of usedBrowserPaths)
+    directories.delete(browserDirectory);
   for (const directory of directories) {
     browserFetcher.logPolitely('Removing unused browser at ' + directory);
     await removeFolderAsync(directory).catch(e => {});
   }
 
   // 3. Install missing browsers for this package.
-  const myBrowsersToDownload = await readBrowsersToDownload(packagePath, browserNames);
-  for (const browser of myBrowsersToDownload) {
-    await browserFetcher.downloadBrowserWithProgressBar(browsersPath, browser).catch(e => {
-      throw new Error(`Failed to download ${browser.name}, caused by\n${e.stack}`);
+  const mainRegistry = new Registry(PACKAGE_PATH);
```

Comment:
nit: mainRegistry -> myRegistry/thisRegistry?

Reply:
I like `myRegistry`: I had these var names back in the days when I was just starting programming